web/src/components/tasks/business-formation/name/BusinessNameStep.tsx

```diff
@@ -86,6 +86,31 @@ export const BusinessNameStep = (): ReactElement => {
                     onBlurNameField(event.target.value);
                   }}
                 />
+                {state.businessNameAvailability?.status === "UNAVAILABLE" && (
```

Comment:
I was looking around for coverage of this functionality because I was surprised you didn't have to change any test to look for this data on the page

and I don't think we have any?

I was looking in BusinessNameStep I think its a missed test that probably deserves coverage (but can definielty be a post ticket refactor) (also you should look around to make sure I didn't miss it being covered somewhere else in the tree)

Reply:
Great observation, @adidner! We have coverage in `BusinessFormationPaginator` and `BusinessNameStep`, where four  tests failed due to __`TestingLibraryElementError: Unable to find an element by: [data-testid="unavailable-text"]`__. The `data-testid` was on the now deleted `<Alert>` so I added to the `<div>`.